### ISSUE TITLE
Require dev

### ISSUE THREAD

**Issue by dereuromark** [inviqa/spryker-debug#32]
For safety reasons, this debug tool should only used locally in debug mode, right?
Using require-dev then.

**Comment by rocketage** [comment]
It's always good practise to minimise access to sensitive information/services so that's a good default.

However I think it's fine to use this module on production environments if frequent access to config/services is required for investigating issues. The commands only offer a convenient way of doing what you could do anyway with shell access:
- Inspect environment variables to figure out config values
- Access databases via config/environment variables
- Access queues via config/environment variables

For complex environments with multiple server nodes/services I actually think it's safer to use this module because you can be sure you are connecting to the correct service, and viewing the final resolved config.

**Comment by rocketage** [comment]
Good point about considering security. Spryker has a pattern already for this in `ConsoleDependencyProvider` (https://github.com/spryker-shop/b2c-demo-shop/blob/master/src/Pyz/Zed/Console/ConsoleDependencyProvider.php#L349)

So you deploy the module on all environments and then decide which commands to expose on development only environments and which should be available everywhere.
